src/containers/pages/AssigmentMapWrapper/index.tsx

```diff
@@ -52,12 +100,17 @@ const AssignmentMapWrapper = (props: AssignmentMapWrapperProps) => {
     getJurisdictionsFeatures,
     currentParentId,
     rootJurisdictionId,
+    jurisdictionsChunkSize,
   } = props;
   const currentChildIds: string[] = currentChildren.length
```

Comment:
I believe the conditional check for this is unnecessary: `[].map(callback) => []`;